pkgs/development/python-modules/plum_py/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, pytest }:
+
+buildPythonPackage rec {
+  pname = "plum_py";
+  version = "0.4.0";
+  format = "wheel";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version format;
```

Comment:
wheels are discouraged, have you tried packaging from source? https://gitlab.com/dangass/plum/-/tree/master

Reply:
this would also bring access to tests to the package

Reply:
looks pretty trivial to package, only has a setup_requires on setuptools, which should be available through the default `buildPythonPackage` behavior

https://gitlab.com/dangass/plum/-/blob/master/setup.cfg#L5

Reply:
Yup, packaged from source now, with tests.